libs/community/tests/integration_tests/chat_models/test_sparkllm.py

```diff
@@ -1,7 +1,48 @@
+from typing import Any
+
 from langchain_core.messages import AIMessage, AIMessageChunk, HumanMessage
+from langchain_core.prompts import ChatPromptTemplate, HumanMessagePromptTemplate
 
 from langchain_community.chat_models.sparkllm import ChatSparkLLM
 
+_FUNCTIONS: Any = [
+    {
+        "type": "function",
+        "function": {
+            "name": "get_current_weather",
+            "description": "Get the current weather in a given location",
+            "parameters": {
+                "type": "object",
+                "properties": {
+                    "location": {
+                        "type": "string",
+                        "description": "The city and state, e.g. San Francisco, CA",
+                    },
+                    "unit": {"type": "string", "enum": ["celsius", "fahrenheit"]},
+                },
+                "required": ["location"],
+            },
+        },
+    }
+]
+
+
+def test_functions_call_thoughts() -> None:
+    chat = ChatSparkLLM()
+
+    prompt_tmpl = "Use the given functions to answer following question: {input}"
+    prompt_msgs = [
+        HumanMessagePromptTemplate.from_template(prompt_tmpl),
+    ]
+    prompt = ChatPromptTemplate(messages=prompt_msgs)
+
+    chain = prompt | chat.bind(functions=_FUNCTIONS)
```

Comment:
suggest implementing `bind_tools` for consistency with other chat models. see example here:

https://github.com/langchain-ai/langchain/blob/bd4b68cd541719e21a81070a394f724d3600b4a2/libs/partners/mistralai/langchain_mistralai/chat_models.py#L626-L627

Reply:
Waiting for the standard bind_tools() method to be implemented, it's critical for an agengt.
I can play around with gpt-4o, qwen-max, and ERNIE-4.0-8K，if bind_tools() is there, I can play with Spark4.0 Ultra too.